pages/index.md

```diff
@@ -6,38 +6,97 @@ PyCon UK is the annual gathering of the UK Python community and its friends from
 
-* * *
+**_Sold Out_**: we are completely sold out with over 700 people coming. We regret
+that we have no waiting list. If you are a sponsor or a speaker with a query about
+tickets please contact the [organising committee](/contact/)    .
 
-Tickets for the main conference have now sold out, and we're expecting over 700 people at the conference!
-(But there are still a handful of tickets left for the [children's day](https://hq.pyconuk.org/children/orders/new/).)
+We [still have tickets](https://hq.pyconuk.org/children/orders/new/) for the 
+[Children's Day](/education/).
 
 * * *
 
-There will be a heavyweight boxing match in Cardiff over the weekend of the
-conference, and so hotel accommodation in the city has been snapped up much
-more quickly this year than usual.  If you're coming to PyCon UK, *book your
-accommodation now*. Don’t delay any further. If you leave it too late, you may
-find that there is no more affordable accommodation available.  Please see the
-[accommodation page](/accommodation/) for more details.
+## Information for Attendees
 
-* * *
+You can review your talk details and the events and meals you've signed up for at 
+[your profile](https://hq.pyconuk.org/). You'll need to sign in using the email
+address with which you booked your ticket.
+
+### Registration & Schedule
+
+* The conference runs from 9am to 6.30pm each day.
+* Registration on each day opens at 8am but will be available throughout the day.
+* The [Schedule of talks](/schedule/) is available on the PyCon UK website.
+* [Monday](/schedule/#monday) is a day for Sprints: a chance to collaborate on
+  your own or someone else's Python project.
+* The [Conference Dinner](https://hq.pyconuk.org/dinners/conference-dinner/) 
+  is on Friday at the conference venue itself in the Lower Hall. 
+  The bar will be open from 6.30pm and dinner is served at 7.30pm.
+* The Contributors' Dinner is on Sunday at the 
+  [Clink Restaurant](http://theclinkcharity.org/the-clink-restaurants/cardiff-wales/) at [whatever time]
+* There are extra events scheduled after-hours, including a [Code Dojo]
+  organised by the [London Python Code Dojo](https://twitter.com/ldnpydojo) team on Thursday and a 
+  [Games Evening] on Saturday.
+
+### Venue
+
+* PyCon UK takes place at [Cardiff City Hall](http://www.cardiffcityhall.com/find-us)
+* If you need to park nearby because of mobility issues, please [contact us](/contact/) and we'll do our best to help.
+* For other parking, you're advised to pre-book via 
+  [NCP Cardiff](https://www.ncp.co.uk/parking-solutions/cities/cardiff)
+  or make other parking arrangements.
+* [A map] of the conference rooms in the City Hall
+
+### Communications
+
+* The Front Desk will be staffed for Registration and general information
+  throughout the conference. Please ask there if you need any help.
+* During the conference the [PyCon UK Slack channel](https://pyconuk-2017.slack.com) 
+  will be used to share announcements and other information
```

Comment:
Add "Ticketholders will receive an invitation to join in the week before the conference", otherwise we'll get people asking how to join.  (I mean, we'll still get people asking how to join...)

Reply:
Added